PsdReader/Parser.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Collections.Generic;
+
+namespace PsdParser
+{
+    class PsdReader
+    {
+        public const string FileHeaderSignatire = "8BPS";                               //default
+        public const string ImageResourceSignature = "8BIM";                            //default
+        public static Dictionary<short, string> ColorMode= new Dictionary<short, string> //all vars of
+            {                                                                            //available colormode
+               {0,"Bitmap"},
+               {1,"GrayScale"},
+               {2,"Indexed"},
+               {3,"RGB"},
+               {4,"CMYK"},
+               {7,"Multichannel"},
+               {8,"Duotone"},
+               {9,"Lab"}
+
+            };
+        /// <summary>
+        /// Read the header part and fill in some characteristics from it in .txt
+        /// </summary>
+        /// <param name="input">stream of.psd</param>
+        /// <param name="output">stream of.txt</param>
+        public static void ReadHeader(BinaryReader input,StreamWriter output)
```

Comment:
Not sure we should receive both input & output objects, but let's leave it like that for now